cluster-autoscaler/cloudprovider/gce/templates.go

```diff
@@ -37,6 +37,11 @@ import (
 // GceTemplateBuilder builds templates for GCE nodes.
 type GceTemplateBuilder struct{}
 
+// MaxPodsPerNode contains info on max pods per node.
+type MaxPodsPerNode struct {
```

Comment:
Why make this a struct and not just *int64? Do we expect more fields in future?

Reply:
Added it just to be safe. But ya, don't think it is needed